routes.js

```diff
@@ -0,0 +1,30 @@
+const express = require('express');
+const app = express();
+const data = require('./data.json');
+
+app.use(express.json());
+
+
+const get = app.get('/get', (req, res) => {
+    res.send(data);
+});
+
+const post = app.post('/post', (req, res) => {
+    const newData = req.body;
+    console.log(req.body);
+    data.push(newData);
+    res.status(201).json(newData);
+});
+
+const del = app.delete('/:id', (req, res) => {
```

Comment:
Consider using a middleware for error handling. This would help to catch and handle any errors that might occur during the execution of route handlers. [important]